flowkit_jwt_generator/flowkit_jwt_generator/__init__.py

```diff
@@ -7,11 +7,17 @@
     get_all_claims_from_flowapi,
     aggregation_types,
     permissions_types,
+    generate_keypair,
+    load_private_key,
+    load_public_key,
 )
 
 __all__ = [
```

Comment:
Just an observation, but I don't think we _need_ the `__all__` variable in `__init__.py` because we want all the imported names to be available anyway. So I suppose we could delete it to avoid the duplication (I get the sense this also applies to a bunch of modules in flowmachine.) But of course also fine to keep it.